aws-py-oidc-provider/__main__.py

```diff
@@ -0,0 +1,70 @@
+import pulumi
+from pulumi_aws import iam
+import requests
+import subprocess
+import OpenSSL
```

Comment:
You can use the TLS provider to generate certs: https://www.pulumi.com/registry/packages/tls/

Reply:
The steps that use OpenSSL are obtaining Pulumi's OIDC IdP certificate chain and using that to produce its thumbprint (it's not creating a new one). It wasn't clear to me how pulumi_tls would recreate this process, so happy to pair with you on this!